cmd/dfget/app/root.go

```diff
@@ -165,63 +174,61 @@ func initClientLog() error {
 }
 
 func initFlags() {
+	// pass to server
 	flagSet := rootCmd.Flags()
 
 	// url & output
-	flagSet.StringVarP(&cfg.URL, "url", "u", "",
-		"will download a file from this url")
+	flagSet.StringVarP(&cfg.URL, "url", "u", "", "URL of user requested downloading file(only HTTP/HTTPs supported)")
 	flagSet.StringVarP(&cfg.Output, "output", "o", "",
-		"output path that not only contains the dir part but also name part")
+		"Destination path which is used to store the requested downloading file. It must contain detailed directory and specific filename, for example, '/tmp/file.mp4'")
 
 	// localLimit & totalLimit & timeout
 	flagSet.StringVarP(&localLimit, "locallimit", "s", "",
-		"rate limit about a single download task, its format is 20M/m/K/k")
+		"network bandwidth rate limit for single download task, in format of 20M/m/K/k")
 	flagSet.StringVar(&totalLimit, "totallimit", "",
-		"rate limit about the whole host, its format is 20M/m/K/k")
+		"network bandwidth rate limit for the whole host, in format of 20M/m/K/k")
 	flagSet.IntVarP(&cfg.Timeout, "timeout", "e", 0,
-		"download timeout(second)")
+		"Timeout set for file downloading task. If dfget has not finished downloading all pieces of file before --timeout, the dfget will throw an error and exit")
 
 	// md5 & identifier
 	flagSet.StringVarP(&cfg.Md5, "md5", "m", "",
-		"expected file md5")
+		"md5 value input from user for the requested downloading file to enhance security")
 	flagSet.StringVarP(&cfg.Identifier, "identifier", "i", "",
-		"identify download task, it is available merely when md5 param not exist")
-
+		"The usage of identifier is making different downloading tasks generate different downloading task IDs even if they have the same URLs. conflict with --md5.")
 	flagSet.StringVar(&cfg.CallSystem, "callsystem", "",
-		"system name that executes dfget")
+		"The name of dfget caller which is for debugging. Once set, it will be passed to all components around the request to make debugging easy")
```

Comment:
It's not only used for debugging, `supernode` can allow or forbid requests from the `callsystem` in white or black list.

Reply:
Since the white/black functionality is still not enabled. Please let us keep it here.